database/migrations/2020_10_20_152318_set_mac_to_one.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use App\Models\InternetAccess;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class SetMacToOne extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $internet_accesses = InternetAccess::all();
+        foreach ($internet_accesses as $internet_access) {
+            $internet_access->update([
+                'auto_approved_mac_slots' => 1,
+            ]);
+        }
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('one', function (Blueprint $table) {
+            //
+        });
```

Comment:
I think this can be buggy when we want to reverse this (we don't have any table names one).

Reply:
Oh, yeah, it was auto generated.